src/validations.test.js

```diff
@@ -463,4 +464,215 @@ describe('validations', () => {
       expect(result).toEqual({ charLength: true, letter: true, digit: true });
     });
   });
+
+  describe('validateName', () => {
+    it('should accept an empty name', () => {
```

Comment:
Is this ok? Should accept empty names?

Reply:
yes, it's another validation. required validation its responsable for that